config.yml

```diff
@@ -0,0 +1,13 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
```

Comment:
The changes to `config.yml` should be broken out into a separate pull request, everything else looks good